src/snabbkaffe.erl

```diff
@@ -539,7 +540,7 @@ strictly_increasing(L) ->
       lists:foldl(Fun, Init, Rest),
       true;
     [] ->
-      true
```

Comment:
if `[]` is not strictly increasing, how do we justify `[1]` being different ?

Reply:
When this function returns normally, it means the check passed. Otherwise, an exception is thrown. It means the return value is useless, it is always equal `true`.
I am planning to change the semantics of this function, to return `false` when the input list is empty. This will lead to more compact code, for example:

```
MyEvents = ?of_kind(my_kind, Trace),
?assertMatch([_|_], MyEvents), %% Check that some events have been matched, to make sure the testcase is not broken and it does something useful
snabbkaffe:strictly_increasing(MyEvents)
```

Becomes:
```
MyEvents = ?of_kind(my_kind, Trace),
?assert(snabbkaffe:strictly_increasing(MyEvents))
```